### DISCUSSION TITLE
Getting most recently played tracks

### DISCUSSION THREAD

**Discussion by Wibblyw** [Chimildic/goofy#256] [Помощь с алгоритмом] [answered]
Does 

        let recentlyPlayed = RecentTracks.get(300);

always get the most recently played 300 tracks, or is it random selection of 300 from all RecentTracks?

**Comment by Chimildic** [comment, marked as answer]
Read the [docs](https://chimildic.github.io/goofy/#/reference/recenttracks?id=get)

```
Get the tracks of the listening history depending on the parameters. Sort by listening date from fresh to older.
```

**Comment by Wibblyw** [comment]
Thank you, yes, I read the doc on RecentTracks.get(). I did not see a parameter for controlling the tracks it selects (only the number of tracks), hence my question.

Please forgive my inexperience - what I'm trying to achieve is an array for the the 300 most recently listened to tracks. 

In my example I know I can sort recentlyPlayed once I have it.  But will the unsorted array initially contain an (unsorted) list of the 300 most recently played tracks, or will it be a random selection of 300 from the maybe 1000's of played tracks history being collect by Goofy?

**Reply by Chimildic** [reply]
**Sort by listening date from fresh to older.**

**Comment by Wibblyw** [comment]
Thanks
